### ISSUE TITLE
About the validation accuracy in baseline

### ISSUE THREAD

**Issue by ahangchen** [xiaolai-sqlai/gluon-reid#8]
[accuracy](https://github.com/xiaolai-sqlai/gluon-reid/blob/master/baseline/train.py#L116) in baseline module is not the accuracy of all the validation set, but the accuracy of the last batch of validation set.
